### ISSUE TITLE
Allow to send 1 ADA

### ISSUE THREAD

**Issue by roccomuso** [vacuumlabs/adalite#671]
Right now adalite doesn't allow to send ` <= 1 ADA`. But only greater value. 
By protocol 1 ADA should be a valid amount instead!

**Comment by PeterBenc** [comment]
Yes this makes sense. Especially since 1 Ada is also the min value for donation in which case no such tx would pass. Thanks.

**Comment by PeterBenc** [comment]
At first, I thought that we implemented the way you so describe, that each must be greater than 1 ADA instead of greater or equal.
But this line shows otherwise https://github.com/vacuumlabs/adalite/blob/16fd62fea789991a74aa89dd0d29c6c38a1a7548/app/frontend/wallet/shelley/shelley-transaction-planner.ts#L124
I tried sending 1 ADA and everything works as expected. Did you have problem sending 1 ADA, maybe just the change output was too small.

PS: I know, the fee argument is useless here since its always 0, this will be refactored.
